indra/newview/llviewerthrottle.cpp

```diff
@@ -84,7 +84,7 @@ LLViewerThrottleGroup::LLViewerThrottleGroup()
 }
 
 
-LLViewerThrottleGroup::LLViewerThrottleGroup(const F32 settings[])
+LLViewerThrottleGroup::LLViewerThrottleGroup(const F32 settings[TC_EOF])
```

Comment:
Does C++ no longer accept an arbitrary-length array argument?

Reply:
c++ supports it, but in the header we've declared this method as taking a fixed length array, and it is being more strict in checking that the declaration and definition types match.